gpu/src/metal/shaders/test/test_bls12381.h.metal

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+template<typename BLS12381, typename Fp>
+[[kernel]] void add(
+    constant Fp* p [[ buffer(0) ]],
+    constant Fp* q [[ buffer(1) ]],
+    device Fp* result [[ buffer(2) ]]
+)
```

Comment:
We need to find a way to compile these functions only on test builds, as the artifact is later embedded in the binary and bloats it unnecessarily.

We should open an issue about this

Reply:
#372